integration/without_gem_rake_task_test.go

```diff
@@ -77,8 +77,8 @@ func testWithoutGemRakeTask(t *testing.T, context spec.G, it spec.S) {
 
 			Expect(logs).To(ContainLines(
 				MatchRegexp(fmt.Sprintf(`%s \d+\.\d+\.\d+`, settings.Buildpack.Name)),
-				"  Assigning launch processes",
-				"    web: rake",
+				"  Assigning launch processes:",
+				`    web (default): rake `,
```

Comment:
Is the extra space on the end of this line a bug in the emitter?

Reply:
Nah, the emitter only checks that the `args` aren't nil, not if it's empty which is reasonable. I was doing something silly in build, which I have fixed